.github/CODEOWNERS

```
@@ -6,6 +6,7 @@
 *       @lengau
 
 # Documentation owners
+/docs/tutorial/ @medubelko @erinecon @jdkandersson 
```

```suggestion
/docs/tutorial/ @medubelko @erinecon
```
I hate to revert the very change I suggested, but adding David is probably not necessary anymore